falcon/util/misc.py

```diff
@@ -428,3 +429,34 @@ def code_to_http_status(code):
         return getattr(status_codes, 'HTTP_' + str(code))
     except AttributeError:
         return str(code)
+
+
+def deprecated_args(allowed_positional, is_method=True):
+    """Flags a method call with positional args as deprecate
+
+    Args:
+        allowed_positional (int): Number of allowed positional arguments
+        is_method (bool, optional): The decorated function is a method. Will
+          add one to the number of allowed positional args to account for
+          ``self``. Defaults to True.
+    """
+
+    template = (
+        'Calls with{} positional args are deprecated.'
+        ' Specify them as keyword arguments instead'
+    )
+    text = ' more than {}'.format(allowed_positional) if allowed_positional else ''
+    warn_text = template.format(text)
+    if is_method:
```

Comment:
Coverage is complaining about this line because the condition is never false.

Reply:
I'm adding a test